Gordon360/Controllers/RecIMController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Gordon360.Authorization;
+using Gordon360.Models.CCT;
+using Gordon360.Models.CCT.Context;
+using Gordon360.Models.ViewModels;
+using Gordon360.Services;
+using Gordon360.Static.Names;
+using Gordon360.Utilities;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
```

Comment:
Do you use all these packages? If you do not need all and are just copy pasting I would avoid that.